pkg/rancher-desktop/assets/translations/en-us.yaml

```
@@ -273,7 +273,7 @@ application:
     startInBackground:
       label: Start in the background
     windowQuitOnClose:
-      label: Keep running in the background when closing application window
+      label: Quit application when closing application window
```

I think a single mention of "application" will suffice for the description  🙂

```suggestion
      label: Quit when closing application window
```

---

I was thinking about "Quit background service when closing application window", to tie it conceptually to the "Start in background" option, but maybe shorter is better. 😄